charts/rancher-logging/100.1.3+up3.17.7/Chart.yaml

```diff
@@ -0,0 +1,25 @@
+annotations:
+  catalog.cattle.io/auto-install: rancher-logging-crd=match
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/deploys-on-os: windows
+  catalog.cattle.io/display-name: Logging
+  catalog.cattle.io/kube-version: '>=1.16.0-0'
```

Comment:
Update this to `catalog.cattle.io/kube-version: '>= 1.16.0-0 < 1.XX.0'` where XX is supported k8s version +1